src/components/ProfilPage.jsx

```diff
@@ -15,14 +15,15 @@ import {
   CardBdd,
 } from "../styled-components/ProfilPage.jsx";
 import ContactUs from "./ContactUs.jsx";
+import UpdateClient from "./UpdateClientForm.jsx";
 
 export default () => {
   const [clients, setClients] = useState([]);
   const [loading, setLoading] = useState(false);
 
   useEffect(() => {
     setLoading(true);
-    axios.get("http://localhost:3001/client/1").then((res) => {
+    axios.get("http://localhost:3001/client/9").then((res) => {
```

Comment:
This line seems random, it's typically the type of line which will break everything